### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by driesvints**
- Rename fixtures to stubs
- Remove BeatSwitch namespace
- Vastly simplified folder structure
- Various other changes: check individual commits

### CODE REVIEW

File: src/Common/Limit.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    }
+
+    public function value(): ?int
+    {
+        return $this->value;
+    }
+
+    public function isUnlimited(): bool
+    {
+        return $this->value() === self::UNLIMITED;
```
Comment (remipelhate) [inline src/Common/Limit.php]: Reference the value instead of the method.

File: src/Common/Limit.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+     */
+    public static function fromString(string $value): self
+    {
+        $value = (new Keyword(static::class, $value))->value();
+
+        if ($value !== null && ! is_numeric($value)) {
+            throw InvalidCriterionValue::expectedNumeric(static::class);
+        }
+
+        return new static($value ? (int) $value : $value);
```
Comment (remipelhate) [inline src/Common/Limit.php]: This is a final class, so use `self` instead of `static`.

File: src/Keywords/HasKeywords.php [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Distil\Keywords;
+
+interface HasKeywords
```
Comment (remipelhate) [inline src/Keywords/HasKeywords.php:5]: Rename the `it_is_keywordable` specs to `it_has_keywords` as well.

**Review by remipelhate — CHANGES_REQUESTED** [review]

**Comment by driesvints** [comment]
@remipelhate done

File: spec/Common/LimitSpec.php [hunk 4]
```diff
@@ -22,7 +22,7 @@ function it_is_initializable()
         $this->shouldHaveType(Limit::class);
     }
 
-    function it_is_keywordable()
+    function it_has_keywordable()
```
Comment (remipelhate) [inline spec/Common/LimitSpec.php]: `it_has_keywordS`, not `it_has_keywordable` :D

**Review by remipelhate — CHANGES_REQUESTED** [review]

**Review by remipelhate — APPROVED** [review]

**Pull request merged**